Layers/FeatureLayer.gd

```diff
@@ -1,7 +1,7 @@
 extends Layer
 class_name FeatureLayer
 
-# is of type Geodot.GeoRasterLayer
+# is of type Geodot.GeoFeatureLayer
 # TODO: look up how to access classes from gdnative for typing
```

Comment:
(Not directly related to this PR, but just to mention: this will finally be fixed in 4.0!)